script.js

```
@@ -6059,4 +6059,188 @@ function showSection(sectionId) {
+        if (button) {
+            button.addEventListener('click', async function() {
+                const typeStock = document.getElementById('type-stock') ? document.getElementById('type-stock').value : '';
+                const date = document.getElementById('date-inventaire') ? document.getElementById('date-inventaire').value : '';
+
+                console.log('=== DEBUG: Button clicked ===');
+                console.log('typeStock:', typeStock);
+                console.log('date:', date);
+                console.log('PRIX_DEFAUT_RANGES:', PRIX_DEFAUT_RANGES);
+
+                // --- STOCK TABLE ---
+                const stockRows = Array.from(document.querySelectorAll('#stock-table tbody tr'));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)